.github/workflows/pipeline-e2e-test.yaml

```diff
@@ -239,4 +324,215 @@ jobs:
         subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
         resource-group: vnet-${{ secrets.RESOURCE_GROUP_TMP }}
         workspace-name: aml-fl2${{ github.run_attempt }}${{ github.run_id }}
-        example: MNIST
\ No newline at end of file
+        example: MNIST
+
+  vnet-sandbox-pneumonia-test:
```

Comment:
general idea: should we reduce the number of tests by assuming that open/vnet are not so much different at this point?

running helloworld on open and vnet makes sense to ensure the setup it working, but maybe not all the examples on both?